### PR TITLE
Feature multibuild results

### PR DESCRIPTION
**Pull request by eduardoj**
Team mob-programming.

### CODE REVIEW

File: src/api/spec/models/package_spec.rb [hunk 1]
```diff
@@ -17,6 +17,23 @@
   let(:other_user) { create(:confirmed_user, login: 'other_user') }
   let(:other_user2) { create(:confirmed_user, login: 'other_user2') }
   let(:other_user3) { create(:confirmed_user, login: 'other_user3') }
+  let(:fake_multibuild_results) do
```
Comment (hennevogel) [inline src/api/spec/models/package_spec.rb]: We don't stub. We use the backend + VCR if we can.

File: src/api/spec/models/package_spec.rb [hunk 2]
```diff
@@ -264,4 +281,44 @@
...
+    it { expect(results_test_package_source.first.architecture).to eq('x86_64') }
+    it { expect(results_test_package_source.first.code).to eq('disabled') }
+    it { expect(results_test_package_source.first.state).to eq('finished') }
+    it { expect(results_test_package_source.first.details).to be_nil }
+
+    it { expect(results_test_package_source.last.repository).to eq('openSUSE_Tumbleweed') }
+    it { expect(results_test_package_source.last.architecture).to eq('x86_64') }
+    it { expect(results_test_package_source.last.code).to eq('unresolvable') }
+    it { expect(results_test_package_source.last.state).to eq('building') }
+    it { expect(results_test_package_source.last.details).to be_nil }
```
Comment (hennevogel) [inline src/api/spec/models/package_spec.rb]: I guess it's okay if you only test the `.first` result. This repetition doesn't help in any way...

File: src/api/spec/models/package_spec.rb (hunk 1 above)
Comment (mdeniz) [inline src/api/spec/models/package_spec.rb]: We don't usually stub... but in this case he has to setup many stuff (interconnect or repos, create a package with a spec, and so on)... Does it worth?

**Comment by mschnitzer** [comment]
I'm fine with it. 👍

File: src/api/spec/models/package_spec.rb (hunk 1 above)
Comment (hennevogel) [inline src/api/spec/models/package_spec.rb]: interconnect is not needed. Just 2 projects and two empty packages.

File: src/api/app/controllers/webui/driver_update_controller.rb [hunk 3]
```diff
@@ -107,7 +107,7 @@ def save
   # TODO: select architecture of binary packages
   def binaries
     required_parameters :repository
-    @binaries = @package.build_result(params[:repository], ['binarylist', 'status']).each('//binary').map{|binary| binary['filename']}
+    @binaries = @package.build_result_old(params[:repository], ['binarylist', 'status']).each('//binary').map{|binary| binary['filename']}
```
Comment (evanrolfe) [inline src/api/app/controllers/webui/driver_update_controller.rb]: Is it possible for this to use the new `@package.buildresults` method? So we can get rid of this `build_result_old` method? Or maybe do that in a card on the next sprint?

Comment (hennevogel) [inline src/api/app/controllers/webui/driver_update_controller.rb]: This is in a dead code path (DuD) that we should remove. So I think we shouldn't invest time into this...

**Comment by hennevogel** [comment]
[(8) Show multibuild result in package view](https://trello.com/c/WcnCtyqZ/101-8-show-multibuild-result-in-package-view)

File: src/api/spec/models/package_spec.rb (hunk 1 above)
Comment (hennevogel) [inline src/api/spec/models/package_spec.rb]: And a running scheduler....

**Comment by hennevogel** [comment]
Please rebase on master.

File: src/api/app/controllers/webui/driver_update_controller.rb (hunk 3 above)
Comment (evanrolfe) [inline src/api/app/controllers/webui/driver_update_controller.rb]: So can `build_result_old` be removed in this PR then? Or create a trello card to remove this?

Comment (Ana06) [inline src/api/app/controllers/webui/driver_update_controller.rb]: A fruit tree card maybe? So we can merge this ASAP

Comment (evanrolfe) [inline src/api/app/controllers/webui/driver_update_controller.rb]: Ok, my main concern is that nobody will end up removing this code if we don't do it now.. But a trello card as long we do get it done soon is fine for me as well.

Comment (eduardoj) [inline src/api/app/controllers/webui/driver_update_controller.rb]: Trello card in fruit tree created: [get rid of build_result_old](https://trello.com/c/LYoHQLqv/214-get-rid-of-build-result-old-in-app-controllers-webui-driver-update-controller-rb)

**Comment by eduardoj** [comment]
[Get rid of build_result_old in app/controllers/webui/driver_update_controller.rb](https://trello.com/c/LYoHQLqv/214-get-rid-of-build-result-old-in-app-controllers-webui-driver-update-controller-rb)

**Pull request merged**